assets/vue/md-textarea.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="input-field col" :class="cols">
+    <textarea lazy v-model="value" class="validate materialize-textarea"
```

Comment:
Honestly I didn't know what lazy is. I started by copying md-input and it must have been in there